lm-cli/lm_cli/subapps/features.py

```diff
@@ -83,7 +86,7 @@ def list_all(
         ),
     )
 
-    formatted_data = format_data(feature_data)
+    formatted_data = format_data(feature_data, flat=True)
```

Comment:
If we're always running `format_data` with `flat=True`, why do we need the flag?

Reply:
I wasn't using the `flat` flag for the `get_one` subcommand (line 125).

Reply:
No longer relevant as I unflattened it in commit 8eb6c76